.vs/ProjectSettings.json

```diff
@@ -0,0 +1,3 @@
+{
+  "CurrentProjectSetting": null
+}
```

Comment:
It seems like the newly created files inside .vs directory are merely local environment set up files that do not need to be included in the repo. I think it's more appropriate for this file to be removed.